system-variables.md

```diff
@@ -1832,6 +1832,15 @@ set tidb_slow_log_threshold = 200;
     - `FAST`: 开启大多数检查项，对性能几乎无影响。
     - `STRICT`: 开启全部检查项，当系统负载较高时，对悲观事务的性能有较小影响。
 
+### `tidb_txn_commit_batch_size` (从 v6.1.0 版本开始引入)
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`16384`
+- 范围：`[1, 1073741824]`
+- 单位：字节
+- 这个变量用于控制 TiDB 向 TiKV 发送的事务提交请求的批量大小。对于业务负载的大部分事务都有大量的写操作，适当调大该变量可以提高批处理的效果。但需要注意的是，设置过大将会超过 TiKV 的 `raft-entry-max-size` 限制，导致提交失败。
```

Comment:
超过 raft-entry-max-size 是否可以取 min 但不报错，不要阻塞业务提交。

Reply:
There is no blocking here. And do we have to go to each tikv and read the configuration every commit? This would have a huge impact on performance, which is the value set by the different components.